forge-gui/res/cardsfolder/v/venture_forth.txt

```diff
@@ -2,8 +2,8 @@ Name:Venture Forth
 ManaCost:3 G
 Types:Sorcery
 K:Suspend:3:1 G
-A:SP$ DigUntil | Defined$ You | Valid$ Permanent.Land | ValidDescription$ land | FoundDestination$ Battlefield | RevealedDestination$ Exile | RememberRevealed$ True | SubAbility$ DBChange | SpellDescription$ Exile cards from the top of your library until you exile a land card. Put that card onto the battlefield and the rest on the bottom of your library in a random order. 
-SVar:DBChange:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | WithCountersType$ TIME | WithCountersAmount$ 3 | SubAbility$ DBRestRandomOrder | SpellDescription$ Exile CARDNAME with three time counters on it.
+A:SP$ DigUntil | Defined$ You | Valid$ Permanent.Land | ValidDescription$ land | FoundDestination$ Battlefield | RevealedDestination$ Exile | RememberRevealed$ True | SubAbility$ DBRestRandomOrder | SpellDescription$ Exile cards from the top of your library until you exile a land card. Put that card onto the battlefield and the rest on the bottom of your library in a random order. 
 SVar:DBRestRandomOrder:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered | Origin$ Exile | Destination$ Library | LibraryPosition$ -1 | RandomOrder$ True | SubAbility$ DBCleanup
```

Comment:
i don't remember exactly what DigUntil can do, but "rest bottom random order", can't this be done as of the DigUntil Effect?

Reply:
It does have that, but here 3 Destinations are involved and the cards need to be exiled first so currently not possible